### DISCUSSION TITLE
ch4-rpc/ch4-01-rpc-intro

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#596] [General]
# ch4-rpc/ch4-01-rpc-intro

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch4-rpc/ch4-01-rpc-intro.html

**Comment by boteman0206** [comment]
为啥我最后一个例子不成功

**Reply by forthespada** [reply]
楼上不成功的可以试试手动关闭当前服务端，然后再启动。。。如果服务开着直接重新跑是会容易不成功

**Reply by jiangchen-123** [reply]
我也不成功，楼上最后有成功吗……

**Reply by jiangchen-123** [reply]
成功了……，似乎是因为之前有一个服务把端口占用了

**Comment by VHcat** [comment]
最后不成功的试试在 print 之前加一个休眠时间，具体原因我也不知道为什么，期待大佬解答。


```go
time.Sleep(time.Second)
```
